src/components/ConferenceList.js

```diff
@@ -2,33 +2,51 @@ import { connect } from 'react-redux';
 import React from 'react';
 import ConferenceCard from './ConferenceCard';
 import SearchBar from './SearchBar';
+import { fetchConferenceData } from '../redux/actions';
 
-const ConferenceList = ({ conferences, searchParam }) => {
-  console.log(searchParam);
-  const filteredList = conferences.filter(
-    item =>
-      item.eventName.toLowerCase().search(searchParam.toLowerCase()) !== -1
-  );
+export class ConferenceList extends React.Component {
+  componentDidMount() {
+    this.props.fetchData();
+  }
 
-  return (
-    <section className=" mw5 mw7-ns center">
-      <h1 className="tl">Upcoming Conferences</h1>
-      <SearchBar />
-      {filteredList &&
-        filteredList.map(conference => (
-          <ConferenceCard key={conference.id} conferenceData={conference} />
-        ))}
-    </section>
-  );
-};
+  render() {
+    console.log(this.props.searchParam);
+    const filteredList = this.props.conferences.filter(
+      item =>
+        item.eventName
+          .toLowerCase()
+          .search(this.props.searchParam.toLowerCase()) !== -1
+    );
+
+    return (
+      <section className=" mw5 mw7-ns center">
+        <h1 className="tl">Upcoming Conferences</h1>
+        <SearchBar />
+        {filteredList &&
+          filteredList.map(conference => (
+            <ConferenceCard key={conference.id} conferenceData={conference} />
+          ))}
+      </section>
+    );
+  }
+}
 
 const mapStateToProps = store => {
   return {
-    conferences: store.mockConferenceData,
+    conferences: store.conferences,
     searchParam: store.searchParam
   };
 };
 
-const ConnectedConferenceList = connect(mapStateToProps)(ConferenceList);
+const mapDispatchToProps = dispatch => {
+  return {
+    fetchData: url => dispatch(fetchConferenceData(url))
```

Comment:
I noticed that the `url` parameter is not being passed into this function when it is used in `componentDidMount()`. If we don't need it, then I think we should remove it.

Reply:
You're right, thanks! I removed that, and pushed again